server/src/test/java/org/elasticsearch/snapshots/SnapshotResiliencyTests.java

```diff
@@ -207,8 +217,12 @@ public void createServices() {
     }
 
     @After
-    public void stopServices() {
-        testClusterNodes.nodes.values().forEach(TestClusterNode::stop);
+    public void stopServices() throws IOException {
+        try {
+            assertNoStaleRepositoryData();
```

Comment:
Can we have two `After` methods - the first one stopServices, the second one - assertNoStaleRepositoryData?
Even if we can have only one after method it probably makes sense to rename it.

Reply:
Renamed it, can't really use another after here since I need to validate consistency before closing the services since that deletes the files.